caikit/core/data_model/base.py

```diff
@@ -152,6 +152,8 @@ def parse_proto_descriptor(mcs, cls):
         # nested messages that have matching names
         cls.full_name = cls._proto_class.DESCRIPTOR.full_name
 
+        # preserve old fields for _make_property_getter later
+        old_fields = cls.fields
```

Comment:
As discussed, we cannot do this because If we add the new fields to `cls.fields,` then `cls.fields` will have `datastream` which will throw errors (since I believe `cls.fields `should only have fields that exist in `proto` world, like `foo` and `bar`)?